test/e2e/mcd_test.go

```diff
@@ -1122,3 +1125,218 @@ func setupForInternalImageRegistryPullSecretTest(t *testing.T, cs *framework.Cli
+
+	mode := 0644
+
+	mc := &mcfgv1.MachineConfig{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "95-worker-install-aeskeyfind-x86-64",
+			Labels: map[string]string{
+				"machineconfiguration.openshift.io/role": "worker",
+			},
+		},
+		Spec: mcfgv1.MachineConfigSpec{
+			Config: runtime.RawExtension{
+				Raw: helpers.MarshalOrDie(ign3types.Config{
+					Ignition: ign3types.Ignition{
+						Version: ign3types.MaxVersion.String(),
+					},
+					Storage: ign3types.Storage{
+						Files: []ign3types.File{
+							helpers.CreateIgn3File(filePath, base64EncodedRPM, mode),
```

Comment:
thought: You don't need to use a MachineConfig to get the RPM onto the node. Instead, you can do something like this: `$ oc cp "$PWD/file.rpm" machine-config-daemon-<id>:/rootfs/tmp/file.rpm -c machine-config-daemon -n openshift-machine-config-operator`

You'll need to identify the correct MCD for the node, but there are helpers for that. Look at how ExecCmdOnNode works to get an idea for that.

Reply:
im going with the MCP route in your previous comment- do you think its MC application related?  just confused where the context deadlines are coming from